SolarTherm/Data/MaterialTables/Sodium.txt

```diff
@@ -0,0 +1,43 @@
+#1
```

Comment:
don't we also have some polynomials for sodium properties somewhere already? just wondering if a lookup table is needed for other reasons, eg convergence?